.circleci/config.yml

```diff
@@ -53,6 +53,8 @@ test_job: &test_job
         command: bundle exec slather coverage -x --scheme $SCHEME ./Kickstarter.xcodeproj
     - codecov/upload:
         file: cobertura.xml
+    - store_artifacts:
+        path: ./Failed_Snapshot_Tests
```

Comment:
Worth noting - this step is in shared code that is run for _all_ our test targets. However, `store_artifacts` is harmless and does nothing if this folder isn't found.